code/modules/mob/living/carbon/human/species/station/skrell/skrell.dm

```diff
@@ -61,8 +61,39 @@
 
 	zombie_type = "Skrell Zombie"
 
+/datum/species/skrell/handle_environment_special(var/mob/living/carbon/human/H)
+	var/old_moisture = H.ST.moisture
+	H.ST.moisture = max(0, H.ST.moisture - rand(-0.8, 1.3))			// Reduce moisture a little.
```

Comment:
rand() cannot do decimals. You will have to do a whole number and divide it. ie: rand(8,13)/10
Also you have -0.8 in there, meaning there is a chance to increase the moisture because your subtracting a negative number, if this worked.

Reply:
Fixed by latest patch